### ISSUE TITLE
Bug about check_anchors

### ISSUE THREAD

**Issue by TaoXieSZ** [ultralytics/yolov5#102]
In the train.py,

Line 202 should be:

`check_anchors(dataset, model=model, thr=hyp['anchor_t'], imgsz=imgsz)`

**Comment by glenn-jocher** [comment]
@ChristopherSTAN thank you noticing this and submitting a bug report! I just pushed a fix for this, should be resolved now. Please git pull and try again.

**Comment by TaoXieSZ** [comment]
@glenn-jocher I feel excited about your Great job! I am training on VOC dataset, and I keep attention on yolov5, thanks again!

**Comment by glenn-jocher** [comment]
Great! Apologies for the bugs, lots of things are changing at the moment, but hopefully it will all smooth out a bit going forward.

**Comment by TaoXieSZ** [comment]
> Great! Apologies for the bugs, lots of things are changing at the moment, but hopefully it will all smooth out a bit going forward.

Another bug there.

If I use `--evolve`, code in line 403 of train.py:
`# Evolve hyperparameters (optional)
    else:
        tb_writer = None
`
Will raise bug

**Comment by glenn-jocher** [comment]
@ChristopherSTAN hmm ok thanks! I will check it out.

**Comment by glenn-jocher** [comment]
Ok, got it fixed now!

**Comment by reactivetype** [comment]
@ChristopherSTAN  did you need to create your own dataloader to work with VOC dataset?

**Comment by TaoXieSZ** [comment]
@reactivetype No, just organize the images and labels as the tutorial showing.

**Comment by glenn-jocher** [comment]
@ChristopherSTAN I haven't trained on VOC myself, but it seems like a commonly used dataset. Perhaps we could add it to the /data folder by giving it it's own voc.yaml and an easy download command?

**Comment by TaoXieSZ** [comment]
@glenn-jocher I am glad to. But I don't have much experiences of using git. So what should I do?

**Comment by glenn-jocher** [comment]
@ChristopherSTAN hmm, well a PR would be easiest (and then you show up as a repo author/contributor). 

I can submit the changes also. All you need to do is upload your voc.yaml here, and some download commands, i.e. a download_voc.sh file also would help. Then I just throw them in the data folder. This might help a lot of people looking for voc training in the future.

**Comment by TaoXieSZ** [comment]
OK, I will check the first way.
Wait a second.
Happy to be listed on your repo LOL.

**Comment by TaoXieSZ** [comment]
@glenn-jocher I just did a PR. 

I am not familiar with git operations. Sorry.

**Comment by glenn-jocher** [comment]
@ChristopherSTAN yeah I see it, thanks for submitting! It looks good, but I'm wondering if there's a way to reduce/merge some of the files. I will look it over more later today or tomorrow.

**Comment by TaoXieSZ** [comment]
@glenn-jocher I think it could. I just copy them from my colab notbook. So they are kind of mess...

**Comment by TaoXieSZ** [comment]
@glenn-jocher Besides, it is the reason why your work means a lot. That is, your model can easily trained on even colab.
